FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentList.cs

```diff
@@ -22,9 +22,50 @@ public unsafe struct AtkComponentList {
     [FieldOffset(0x148)] public AtkCollisionNode* HoveredItemCollisionNode;
     [FieldOffset(0x150)] public int HoveredItemIndex2; // Repeat?
     [FieldOffset(0x158)] public int HoveredItemIndex3; // Repeat?
+    [FieldOffset(0x193)] public bool IsUpdatePending;
+
+    [VirtualFunction(25)]
+    public partial AtkComponentListItemRenderer* GetItemRenderer(int index);
+
+    [VirtualFunction(26)]
+    public partial void SetItemDisabledState(int index, bool disabled);
+
+    [VirtualFunction(27)]
+    public partial bool GetItemDisabledState(int index);
+
+    [VirtualFunction(28)]
+    public partial void SetItemHighlightedState(int index, bool highlighted, bool triggerUpdate = true);
+
+    [VirtualFunction(29)]
+    public partial bool GetItemHighlightedState(int index);
+
+    [VirtualFunction(31)]
+    public partial void SelectItem(int index, bool dispatchEvent37 = false);
+
+    [VirtualFunction(32)]
+    public partial void DeselectItem();
+
+    [VirtualFunction(36)]
+    public partial int GetItemCount();
+
+    /// <remarks> Used by <see cref="AtkComponentDropDownList"/>. </remarks>
+    [MemberFunction("E8 ?? ?? ?? ?? 45 38 A4 3E"), GenerateCStrOverloads]
+    public partial void SetItemLabel(int index, byte* text);
+
+    /// <remarks> Used by <see cref="AtkComponentDropDownList"/>. </remarks>
+    [MemberFunction("E8 ?? ?? ?? ?? 89 5C 24 28 48 8B 5C 24")]
+    public partial byte* GetItemLabel(int index);
```

Comment:
Would you be willing to add a helper to get the item label as a string?

Reply:
I could, but we have `MemoryHelper.ReadStringNullTerminated()`? 🤔

Reply:
Normally we can use the attribute to generate string overloads, but we can't do that for a return value, it's just a ease-of-use thing. Up to you.

Reply:
Hm, nah. :rofl: I don't really like to make a mess here.

Maybe with the next API bump we could think about replacing all byte* with a little helper struct for null terminated strings, so we can still pass them as pointers but also have a C# ToString method. 👀